examples/hub-values.yaml

```diff
@@ -23,10 +28,6 @@ etcd:
   # storageSize is configured per workspace.
   storageSize: "3Gi"
 
-loadBalancer:
-  # ip is configured per workspace.
-  ip: "192.0.2.3"
-
 pachd:
   postgresql:
     host: "169.254.169.254"
```

Comment:
169.254.0.0/16 is reserved by [RFC 3827](https://datatracker.ietf.org/doc/html/rfc3927) for link-local addresses. In order to prevent collisions with actual link-local interfaces in real-world clusters, I think it would be best to use a block reserved by [RFC 5737](https://datatracker.ietf.org/doc/html/rfc5737) specifically for documentation and implicitly for testing (e.g. 192.0.2.0/24).